src/Atlas2DSlice.ts

```diff
@@ -147,15 +169,11 @@ export default class Atlas2DSlice implements VolumeRenderImpl {
 
       this.setUniform("AABB_CLIP_MIN", bounds.bmin);
       this.setUniform("AABB_CLIP_MAX", bounds.bmax);
-      const slice = Math.floor(this.settings.zSlice);
-      const sizez = this.volume.imageInfo.volumeSize.z;
-      if (slice >= 0 && slice <= sizez - 1) {
-        this.setUniform("Z_SLICE", slice);
-        const sliceRatio = slice / sizez;
-        this.volume.updateRequiredData({
-          subregion: new Box3(new Vector3(0, 0, sliceRatio), new Vector3(1, 1, sliceRatio)),
-        });
-      }
```

Comment:
The bounds checking that was originally done here is now done in `updateSlice`, but it doesn't stop the call to `this.volume.updateRequiredData`... Should `updateSlice` have a return value to indicate if the slice is in range?

Reply:
Yep - whoops! I remember making a mental note that I'd have to preserve that behavior when factoring out updates into their own function, but I missed it when I circled back and actually did it.